server/service/osquery_utils/queries.go

```diff
@@ -1373,18 +1388,21 @@ func directIngestDiskEncryption(ctx context.Context, logger log.Logger, host *fl
 	return ds.SetOrUpdateHostDisksEncryption(ctx, host.ID, encrypted)
 }
 
-func directIngestDiskEncryptionKeyDarwin(
+func directIngestDiskEncryptionKeyFileDarwin(
 	ctx context.Context,
 	logger log.Logger,
 	host *fleet.Host,
 	ds fleet.Datastore,
 	rows []map[string]string,
 ) error {
+	// TODO: What should happen if file does not exist? The query will return an error (because the extension table
+	// failed to generate). Should we be updating the datastore to delete the prior key (if any)?
```

Comment:
That's a good question, do we know how the `file_lines` table handled that condition? Did it result in an empty result or an error?